src/convertToObject.js

```diff
@@ -1,18 +1,25 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssStyles = {};
+
+  const redactedString = sourceString
+    .split(';')
+    .filter(s => s.trim().length > 0)
```

Comment:
```suggestion
    .filter(s => s.trim().length)
```
If `s.trim().length === 0` it's already false value. So other cases are true.